src/dialogs/quick-open/ha-quick-open-dialog.ts

```diff
@@ -109,15 +107,17 @@ export class QuickOpenDialog extends LitElement {
     );
 
     return html`
-      ${entities.map((entity_id) => {
-        const domain = computeDomain(entity_id);
-        return html`
-          <mwc-list-item @click=${this._entityMoreInfo} graphic="icon">
-            <ha-icon .icon=${domainIcon(domain)} slot="graphic"></ha-icon>
-            ${entity_id}
-          </mwc-list-item>
-        `;
-      })}
+      <mwc-list activatable @selected=${this._entityMoreInfo}>
+        ${entities.map((entityId) => {
+          const domain = computeDomain(entityId);
+          return html`
+            <mwc-list-item graphic="icon" .entityId=${entityId}>
+              <ha-icon .icon=${domainIcon(domain)} slot="graphic"></ha-icon>
+              ${entityId}
```

Comment:
We should not show the Entity ID as the main value. (I know this wasn't changed in this PR)

We should show the Friendly name and the Entity ID as Secondary under that. MWC list has this ability built in.

Reply:
Yeah I saw that in the docs. Dig it.

Reply:
Actually... I'm not sure I agree. 

My worry is that by having the primary text be the friendly name, folks will think they need to filter using the friendly name rather than the entity id.

Perhaps we could surface the friendly name on the right-hand side of the list item?

(Also, I was kinda thinking we could use secondary field for state information further down the road)

Reply:
It should search both.

Reply:
We should not show users the entity id
Thats too technical IMO
though it will only be used by power users 🤔 

Entity ID may not be what they need to search

Example: ZHA device entities can have crazy ids. I rename the friendly name not the id.

We should use the friendly name before the entity id

Reply:
Okay. You've convinced me. Is it too early to start submitting feature request issues for tracking ;)

Reply:
Nope. Its a good way to help yourself track as well as get opinions and upvotes from others

Reply:
Here's a PR for showing the friendly name, btw: https://github.com/home-assistant/frontend/pull/7306